src/bar.scss

```diff
@@ -72,6 +72,11 @@ $block: #{$fd-namespace}-bar;
     max-width: 100%;
     max-height: 100%;
 
+    @include fd-focus() {
+      outline: var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
+      outline-offset: var(--sapContent_FocusWidth);
+    }
+
```

Comment:
`fd-bar__element` is just a wrapper whose sole purpose is to apply some spacing on the right hand side (margin-right: $fd-bar-element-spacing;). This element SHOULD NOT get a focus. What gets the focus is the element inside it.

Reply:
these focus change is not required